src/ngx_http_v2_push_module.c

```diff
@@ -723,7 +725,54 @@ ngx_http_v2_push_populate_path(ngx_http_request_t *r, u_char *u_str, size_t u_le
         return NGX_ERROR;
     }
 
-    return ngx_http_process_request_uri(r);
+    if (r->args_start) {
```

Comment:
this is basically just copied from nginx with the `ngx_http_close_request` and `ngx_http_finalize_request` removed?